src/redteam/constants.ts

```diff
@@ -32,6 +38,7 @@ export const ALL_PLUGINS = [...new Set([...DEFAULT_PLUGINS, ...ADDITIONAL_PLUGIN
 // Duplicated in src/web/nextui/src/app/report/constants.ts for frontend
 export const subCategoryDescriptions: Record<string, string> = {
   'excessive-agency': 'Model taking excessive initiative or misunderstanding its capabilities.',
+  harmful: 'All harmful categories',
```

Comment:
Including this results in a `harmful` description when running `promptfoo redteam init`.